<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainMenu</class>
 <widget class="QDialog" name="MainMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>313</width>
    <height>212</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushVolume">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Set Volume</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushMap">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Key Mapping</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushQuit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>140</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Quit Game</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushOk">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
